src/components/Bars/barHelper.ts

```diff
@@ -1,4 +1,4 @@
-import { ScaleBand, ScaleLinear } from "d3-scale";
+import { ScaleBand, ScaleLinear, scaleOrdinal } from "d3-scale";
```

Comment:
Is this an unused var? I can see its been added but not where it has been used.

Reply:
Yes its unused, thought I needed it then I didn't! Thanks - updated